components/workflow.rst

```diff
@@ -19,6 +19,87 @@ You can install the component in 2 different ways:
 * :doc:`Install it via Composer </components/using_components>` (``symfony/workflow`` on `Packagist`_);
 * Use the official Git repository (https://github.com/symfony/workflow).
 
-For more information, see the code in the Git Repository.
+.. include:: /components/require_autoload.rst.inc
+
+Creating a Workflow
+-------------------
+
+The workflow component gives you an object oriented way to define a process
```

Comment:
The more I read see, the more I think we should rename - at least in docs - `Workflow` component to `Workflow Engine` component. Am I crazy ? /c @Nyholm